src/componentTest/resources/expectedTemplateContent/performance-tests/performance-tests/src/main/scala/com/blackbaud/service/ServiceTest.scala

```diff
@@ -0,0 +1,75 @@
+package com.blackbaud.service
+
+import com.blackbaud.gatling.bbauth.BBAuthHelper
+import com.blackbaud.gatling.SimulationConfiguration
+import io.gatling.core.Predef.{exec, forever, scenario}
+import io.gatling.http.Predef.http
+import io.gatling.core.Predef._
+import io.gatling.core.structure.{ChainBuilder, ScenarioBuilder}
+import io.gatling.http.protocol.HttpProtocolBuilder
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class ServiceTest extends SimulationConfiguration {
```

Comment:
Consider just doing something like `new SimulationConfiguration(userCount = 6, loadUsersOverSec = 2, maxTestRuntimeSec = 300)` or new `SimulationConfiguration()` for the defaults instead of extending it.